Geometry/ForwardCommonData/data/pixfwd.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
```

Comment:
This file should have a version number in the directory name, maybe `Geometry/ForwardCommonData/data/forward/2021/v1/pixfwd.xml`?